packages/datadog_tracking_http_client/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: datadog_tracking_http_client
 description: A wrapping implementation of HttpClient for tracking resources with Datadog
-version: 1.1.0-rc.2
+version: 1.2.0
```

Comment:
Shouldn't this be `1.1.0`?

Reply:
This is the bump post release of 1.1 (this is the tracking client)